resources/assets/sass/app.scss

```diff
@@ -411,5 +411,5 @@ table.vuetable {
   position: fixed;  
   left:30%;
   right:30%;
-
+  z-index: 1051;
```

Comment:
wouldn't it better to have a more meaningful number like 9999 and set it as the maximum to be used for the z-index of all our system?

Reply:
Change to sass variable: $alertZIndex